onnxruntime/test/providers/cpu/math/einsum_test.cc

```diff
@@ -254,6 +254,16 @@ TEST(Einsum, ExplicitEinsumAsMatmul_2) {
   test.Run();
 }
 
+TEST(Einsum, ExplicitEinsumAsMatmul_3) {
+  OpTester test("Einsum", 12, onnxruntime::kOnnxDomain);
```

Comment:
There were a lot of test cases in this WIP PR: [https://github.com/microsoft/onnxruntime/pull/7374/files](https://github.com/microsoft/onnxruntime/pull/7374/files)

Could/should we include at least some of those?

Reply:
Added all of them